src/rules/enforce-firestore-facade.ts

```
@@ -308,9 +308,33 @@ const isFirestoreMethodCall = (node: TSESTree.CallExpression): boolean => {
       return false;
     }
 
+    // Check if it's a BatchManager or other custom management class
+    if (name.includes('Manager') || name.includes('BatchManager')) {
+      return false;
+    }
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)